python3/packages/observer.py

```diff
@@ -41,7 +41,7 @@
 # We only want to import opentelemetry libraries if instrumentation is enabled
 # pylint: disable=import-outside-toplevel
 
-DEBUG_ENABLED = False
+DEBUG_ENABLED = os.getenv("OBSERVER_DEBUG")
```

Comment:
Another env variable is used for detecting the code is run in a test environment: `XAPI_TEST`